src/components/pages/register.yml

```diff
@@ -10,12 +10,18 @@ emailIsInvalid:
 emailIsInUse:
   en: The email address is already registered
   de: Die E-Mail-Adresse ist bereits vergeben
+usernameIsTooShort:
+  en: The user name must be at least {length} characters long
+  de: Der Benutzername muss mindestens {length} Zeichen lang sein
 usernameIsInUse:
   en: The user name is already registered
   de: Der Benutzername ist bereits vergeben
 enterPassword:
   en: Please enter a password
   de: Bitte geben Sie ein Kennwort an
+passwordIsInvalid:
+  en: The password must be at least {length} characters long and contain at least one letter and one number
+  de: Das Kennwort muss mindestens {length} Zeichen lang sein ...
```

Comment:
The german is lacking some details about the one letter and one numner.

Reply:
Yup, was on its way